### PR TITLE
DSC 4 cards + moving upcoming to folders

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/my_champion_stands_supreme.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:I Am Duskmourn
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/my_champion_stands_supreme.txt:1]: wrong name?

File: forge-gui/res/cardsfolder/upcoming/valgavoth_harrower_of_souls.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Valgavoth, Harrower of Souls
+ManaCost:2 B R
+Types:Legendary Creature Elder Demon
+PT:4/4
+K:Flying
+K:Ward:PayLife<2>
+T:Mode$ LifeLost | ValidPlayer$ Opponent | TriggerZones$ Battlefield | FirstTime$ True | Execute$ TrigPutCounter | TriggerDescription$ Whenever an opponent loses life for the first time during each of their turns, put a +1/+1 counter on CARDNAME and draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/valgavoth_harrower_of_souls.txt:7]: valid incomplete

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/valgavoth_harrower_of_souls.txt:7]: Opponent.Active ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/valgavoth_harrower_of_souls.txt:7]: yea

**Comment by Hanmac** [comment]
@tehdiplomat @tool4ever @Simisays i would recomment to do the Migrate in a Seprarate MR

**Review by tool4ever — APPROVED** [review]
I think we can move it now this time before it gets too bloated 🤷‍♂️

**Review by Hanmac — APPROVED** [review]

**Pull request merged**